lms/djangoapps/grades/tests/test_transformer.py

```diff
@@ -330,6 +330,16 @@ def test_course_version_collected_in_split(self):
         block_structure = get_course_blocks(self.student, blocks[u'course'].location, self.transformers)
         self.assertIsNotNone(block_structure.get_xblock_field(blocks[u'course'].location, u'course_version'))
 
+    def test_grading_policy_collected(self):
+        blocks = self.build_course_with_problems()
+        block_structure = get_course_blocks(self.student, blocks[u'course'].location, self.transformers)
+        self.assert_collected_transformer_block_fields(
+            block_structure,
+            blocks[u'course'].location,
+            self.TRANSFORMER_CLASS_TO_TEST,
+            grading_policy_hash=-5366073452054513847,
```

Comment:
@efischer19 good idea. I'll include it as a method variable.

As far as changing the grading policy and re-calculating, that would be really valuable and I'll look into adding that to the test!